pkg/systemd/terminal.jsx

```diff
@@ -81,8 +95,35 @@ const _ = cockpit.gettext;
         }
 
         async componentDidMount() {
+            cockpit.addEventListener("locationchanged", this.onNavigate);
             const user = await cockpit.user();
-            this.setState({ user, channel: this.createChannel(user) });
+            var dir
```

Comment:
*jedi wave* `var` is not the keyword you are looking for

I.e. use `let dir;`.

Please run `test/static-code`, it [includes that issue and lots of others](https://github.com/cockpit-project/cockpit/actions/runs/11587590817/job/32260000860?pr=21162#step:2:475).